### PR TITLE
Update manage-feedback-product-insights.md

### PR DESCRIPTION
**Pull request by msbemba**
checked and updated admin center to reflect M365 admin center and also added a hyper link to the same

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8967

File: microsoft-365/admin/manage/manage-feedback-product-insights.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ You need to be an [administrator](../add-users/about-admin-roles.md) to view and
 
 ## NPS survey insights
 
-1. In the admin center, go to the **Health** > **Product feedback** > **NPS survey insights**.
+1. In the [M365 admin center](https://admin.microsoft.com/), go to the **Health** > **Product feedback** > **NPS survey insights**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-feedback-product-insights.md]:
```suggestion
1. In the [Microsoft 365 admin center](https://admin.microsoft.com/), go to **Health** > **Product feedback** > **NPS survey insights**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**